tests/request_animation_frame.cpp

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2015 The Emscripten Authors.  All rights reserved.
+ * Emscripten is available under two separate licenses, the MIT license and the
+ * University of Illinois/NCSA Open Source License.  Both these licenses can be
+ * found in the LICENSE file.
+ */
+
+// Test RAF is actually used (and not setTimeout etc.)
```

Comment:
It looks like this test verifies that requestAnimationFrame is used for scheduling on the main thread. How does it test that it also works on workers? Is it run by the test harness in multiple modes?